### PR TITLE
feat: UUID v7 support

### PR DESCRIPTION
**Pull request by ssaarela**

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -540,8 +540,8 @@ Unless otherwise stated, all validators require non-null and non-undefined value
...
 | notEmpty                |                                                                  | Requires non-null, non-undefined and not empty. Works with anything having numeric `length` property, e.g. string or array.                |
 | notBlank                |                                                                  | Requires non-null, non-undefined and not blank (whitespace only string) values.                                                            |
-| uuid                    | version?: number                                                 | Uses `uuid-validate` package to validate input.                                                                                            |
-| pattern                 |  pattern: string \| RegExp, flags?: string                       | Tests input against the pattern.                                                                                                           |
+| uuid                    | version?: number                                                 | Uses `uuid` package to validate input.                                                                                                     |
```
Comment (karlkeskyla) [inline packages/core/README.md:543]: Trim trailing spaces setting could be missing 🤔

Comment (ssaarela) [reply packages/core/README.md:543]: It's table formatting, making the columns in all the rows same size. Row ends with | :)

**Review by karlkeskyla — COMMENTED** [review]
Neat stuff!

Added a tiny comment with a question

**Review by karlkeskyla — APPROVED** [review]
Neat!

**Pull request merged**